README.md

```diff
@@ -313,3 +314,50 @@ vagrant destroy
 ```bash
 vagrant ssh
 ```
+
+# Helper scripts
+
+`concent-env.sh`
```

Comment:
Before merging this please remind me to add a bit more description for these scripts.